### ISSUE TITLE
demo代码没有正确输出

### ISSUE THREAD

**Issue by leslieliu-cn** [TW-NLP/ChineseErrorCorrector#2]
`# pip install transformers
from transformers import AutoModelForCausalLM, AutoTokenizer
checkpoint = "./ChineseErrorCorrector2-7B"

device = "cpu" #cuda for GPU usage or "cpu" for CPU usage
tokenizer = AutoTokenizer.from_pretrained(checkpoint)
model = AutoModelForCausalLM.from_pretrained(checkpoint).to(device)

input_content = "你是一个文本纠错专家，纠正输入句子中的语法错误，并输出正确的句子，输入句子为：\n少先队员因该为老人让坐。"

messages = [{"role": "user", "content": input_content}]
input_text=tokenizer.apply_chat_template(messages, tokenize=False)

print(input_text)

inputs = tokenizer.encode(input_text, return_tensors="pt").to(device)
outputs = model.generate(inputs, max_new_tokens=1024, temperature=0, do_sample=False, repetition_penalty=1.08)

print(tokenizer.decode(outputs[0]))`

ChineseErrorCorrector git:(main) ✗ uv run hello.py                                                 
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:24<00:00,  6.01s/it]
<|im_start|>system
You are Qwen, created by Alibaba Cloud. You are a helpful assistant.<|im_end|>
<|im_start|>user
你是一个文本纠错专家，纠正输入句子中的语法错误，并输出正确的句子，输入句子为：
少先队员因该为老人让坐。<|im_end|>

/Users/leslie/projects/pyprojects/ChineseErrorCorrector/.venv/lib/python3.11/site-packages/transformers/generation/configuration_utils.py:629: UserWarning: `do_sample` is set to `False`. However, `temperature` is set to `0` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `temperature`.
  warnings.warn(
/Users/leslie/projects/pyprojects/ChineseErrorCorrector/.venv/lib/python3.11/site-packages/transformers/generation/configuration_utils.py:634: UserWarning: `do_sample` is set to `False`. However, `top_p` is set to `0.8` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `top_p`.
  warnings.warn(
/Users/leslie/projects/pyprojects/ChineseErrorCorrector/.venv/lib/python3.11/site-packages/transformers/generation/configuration_utils.py:651: UserWarning: `do_sample` is set to `False`. However, `top_k` is set to `20` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `top_k`.
  warnings.warn(
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

**Comment by leslieliu-cn** [comment]
只是慢，等一会出结果了

**Comment by TW-NLP** [comment]
您好，可以参考最新的推理实例。
